src/processor/graph.monitor.processor.service.ts

```diff
@@ -0,0 +1,136 @@
+import { InjectRedis } from '@liaoliaots/nestjs-redis';
+import { InjectQueue, Processor, WorkerHost } from '@nestjs/bullmq';
+import { Injectable, Logger } from '@nestjs/common';
+import { Job, Queue } from 'bullmq';
+import Redis from 'ioredis';
+import { MILLISECONDS_PER_SECOND } from 'time-constants';
+import { RegistryError } from '@polkadot/types/types';
+import { BlockchainService } from '#app/blockchain/blockchain.service';
+import { BlockchainConstants } from '#app/blockchain/blockchain-constants';
+import { ITxMonitorJob } from '#app/interfaces/monitor.job.interface';
+import { IGraphUpdateJob } from '#app/interfaces/graph-update-job.interface';
+
+@Injectable()
+@Processor('graphTxMonitorQueue')
+export class GraphNotifierService extends WorkerHost {
```

Comment:
nit: filename should match class name, ie `graph-notifier.service.ts`, if we're following the pattern elsewhere in the repo